product_release_post.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# vi: set ft=python :
+import requests
+import markdown
+
+products_map = {
+    "NMO": "node-maintenance-operator",
+    "NHC": "node-healthcheck-operator",
+    "SNR": "self-node-remediation",
+    "FAR": "fence-agents-remediation",
+    "MDR": "machine-deletion-remediation",
+}
```

Comment:
This was an experiment, but I don't see anymore any benefit in having this map. I will just use the full name when necessary